packages/cloudflare/src/cli/build/bundle-server.ts

```diff
@@ -139,6 +140,41 @@ globalThis.__dangerous_ON_edge_converter_returns_request = true;
   console.log(`\x1b[35mWorker saved in \`${openNextServerBundle}\` 🚀\n\x1b[0m`);
 }
 
+/**
+ * The `loadInstrumentationModule` method (source: https://github.com/vercel/next.js/blob/5b7833e3/packages/next/src/server/next-server.ts#L301)
+ * calls `module.findSourceMap` (https://nodejs.org/api/module.html#modulefindsourcemappath) which we haven't implemented causing a runtime error.
+ *
+ * To solve this issue this function gets all the `loadInstrumentationModule` declarations found in the file and removes all the statements
+ * from their bodies (making them no-op methods).
+ *
+ * Instrumentation is a Next.js feature for monitoring and logging (see: https://nextjs.org/docs/app/building-your-application/optimizing/instrumentation),
+ * the removal of this method's logic most likely breaks this feature (see: https://nextjs.org/docs/app/api-reference/file-conventions/instrumentation),
+ * so this function is likely temporary and something that we'll have to fix in the future.
+ *
+ * TODO: investigate and re-enable instrumentation (https://github.com/opennextjs/opennextjs-cloudflare/issues/160)
```

Comment:
Can you please create a separate issue for this todo and add the link here. 
We can close the current issue when we merge this to main but keep the new one for further investigation. 

You can optionally move this function to the patch folder. 

Thanks

Reply:
> Can you please create a separate issue for this todo and add the link here.
> We can close the current issue when we merge this to main but keep the new one for further investigation.

Done 👍 

> You can optionally move this function to the patch folder.

oh yes! sorry I forgot how our files structure looks like! done! 🙇 👍